social-network/src/ArticleItem.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class ArticleItem extends React.Component {
+    
+    static propTypes = {
+        article: PropTypes.object
```

Comment:
This will work but be sure to review how to use shape to dig into an object and define more of its keys using proptypes as in practice you will often be defining props more strictly.